pkg/plugins/constants.go

```diff
@@ -0,0 +1,11 @@
+package plugins
+
+import "time"
+
+const (
+	// DefaultTimeout is the default timeout for gRPC operations
+	DefaultTimeout = 30 * time.Second
+
+	// DefaultPluginDir is the default directory where plugins are stored
+	DefaultPluginDir = "/usr/local/lib/lima/plugins"
```

Comment:
This should be probably like `PREFIX/libexec/lima/plugins/vm`.

The PREFIX must be detected via the `os.Executable` path
https://github.com/lima-vm/lima/blob/0625d0b084450e874869dcbc9f63d4312797c3fe/pkg/usrlocalsharelima/usrlocalsharelima.go#L41-L43

Reply:
Alternatively we can look for `lima-plugin-vm-<PLUGIN>` in `PATH`.
This model is similar to containerd runtime plugins. (`containerd-shim-runc-v2`)